src/dart-2.md

```diff
@@ -1,8 +1,47 @@
 ---
-title: Dart 2 Migration
+title: Dart 2
 description: How Dart 2 is different from Dart 1.x, and how you can convert your code to work with Dart 2.
 ---
 
+Dart 2 has a few key differences from earlier versions of Dart.
```

Comment:
By "version" we usually mean major version (IMHO), so "from earlier versions" seems strange to me.

> Dart 2 has a few key differences from Dart 1.x.